modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
Do you have any suggestion on how to proceed here towards (enforcing) a standardized binding layout?